python_modules/libraries/dagster-databricks/dagster_databricks/utils.py

```diff
@@ -0,0 +1,25 @@
+"""
+TODO: this is copied from `dagster_aws.emr` and should be shared instead somehow.
```

Comment:
It would be good to share it - perhaps it makes sense inside the dagster_pyspark package?

Reply:
Yep agreed. I've moved it there and changed dagster_emr to point to it.